src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +332,30 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    private static String getDockerUserAndGroup() {
+        String user = executeBashCommand("id -u");
+        String group = executeBashCommand("id -g");
+        return String.format("%s:%s", user,group);
+    }
+
+    private static String executeBashCommand(String command) {
+        StringBuilder output = new StringBuilder();
+        try {
+            String line = "";
+            Process proc = Runtime.getRuntime().exec(command);
+            // Read the output
+            BufferedReader reader =
```

Comment:
Use try-with-resource for this `reader` (preferred), or close it explicitly.

Reply:
I closed the buffer in order to avoid nested try